tool.gpt

```diff
@@ -3,22 +3,66 @@ Description: Retrieve information from files uploaded by the user.
 Type: context
 Share Tools: Knowledge Retrieval
 Share Context: github.com/gptscript-ai/context/workspace
+Share Context: Knowledge Retrieval Context with *
 
 #!sys.echo
 
 You have access to a RAG tool named "Knowledge Retrieval".
 It will work with files from the workspace it previously ingested on behalf of the user.
-Use it to answer questions from the user and ALWAYS give a proper citation to the best of your abilities, including the source references (filename, page, etc.).
+Use it to answer questions from the user.
+Give citations or source references only if asked for it.
```

Comment:
Not sure how useful this will be but wdyt of this?
```suggestion
Give citations or source references only if asked for it and that doesn't conflict with any other instructions you've received in your system prompt.
```

Reply:
Absolutely worth it until we have a codified setting for this :+1: